DOCKER.md

```diff
@@ -20,7 +20,7 @@ cd create-dev-env`
 
 To get started, create your docker image:
 ```bash
-docker build -t lisk-sdk --build-arg user_name=liskjs --build-arg user_id=$UID .
+docker build -t lisk-sdk --build-arg user_id=$UID .
```

Comment:
What's the reason the `Dockerfile` does not derive `FROM nodejs`? (it would build in a couple seconds instead of minutes)

Reply:
Yeah, I think it should.

The original incarnation of this was more of an explicit "here's everything that it's doing" thing and trying to be very minimal.